src/Transport/Topology/ICreateTopology.cs

```diff
@@ -7,8 +7,15 @@ namespace NServiceBus.Transport.AzureServiceBus
 
     // note there is some creation logic elsewhere already, those calls should be removed and centralized here
 
+    [ObsoleteEx(Message = "Internal contract that shouldn't be exposed.", TreatAsErrorFromVersion = "8.0", RemoveInVersion = "9.0")]
     public interface ICreateTopology
     {
         Task Create(TopologySection topology);
     }
+
+    // TODO: Move into internalized ICreateTopology in v8
+    interface ICreateTopologyAbleToDeleteSubscriptions
```

Comment:
Why is the delete subscription contained in this PR? I suppose those changes are unrelated to this PR?